src/frontend/shared-components/src/terms-of-use/TermsOfUse.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FaPrint } from "react-icons/fa";
+import { Button } from "../button/Button";
+
+import "./TermsOfUse.css";
+
+export default function TermsOfUse({
+  onScroll,
+  acceptTerms,
+  continueButton,
+  cancelButton,
+  content
```

Comment:
Should we validate that `content` is valid html ?

Reply:
I'm not too sure thats an interesting question. But I dont think that should be responsibility of this component because what should happen if it isnt valid? react will throw an error anyways if the consumer tries to pass something invalid. the consumer just needs to ensure that they are sending a valid object